WordPress/Classes/ViewRelated/Post/PostListViewController.swift

```diff
@@ -25,6 +25,10 @@ class PostListViewController : AbstractPostListViewController, UIViewControllerR
     @IBOutlet private var imageCellForLayout: PostCardTableViewCell!
     @IBOutlet private weak var authorFilterSegmentedControl: UISegmentedControl!
 
+    @IBOutlet var authorsFilterView : UIView! // Search lives here on iPad
```

Comment:
This is an old comment left over from `WPSearchController` – I don't think it makes any sense, so removing.